K2Bridge/Controllers/IndexListController.cs

```diff
@@ -43,5 +45,18 @@ public async Task<IActionResult> Process(string indexName)
 
```

Comment:
This method can be deleted:
public async Task<IActionResult> Process(string indexName) 

(verified - works without it)

Reply:
Removed.